conjur/init_logic.py

```diff
@@ -0,0 +1,119 @@
+# -*- coding: utf-8 -*-
+
+"""
+InitLogic module
+
+This module is the business logic for writing configuration information
+to the user's machine as well as fetching certificates from Conjur
+"""
+
+# Builtins
+import logging
+import sys
+import os.path
+
+# Third party
+import socket
```

Comment:
Builtins

Reply:
This file has
```
# Builtins
import logging
import sys
import os.path

# Third party
import socket
import yaml
from OpenSSL import SSL
from OpenSSL.crypto import FILETYPE_PEM
```

Should only socket be a built in or the whole Third party list should be Builtins?